packages/vrm-viewer/rollup.config.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-env node */
```

Comment:
We should also avoid duplicating rollup config everywhere or swtich to something like esbuild when possible

Reply:
Addressed @ #1411 🎉